src/main/java/team05/integrated_feed_backend/module/post/entity/Post.java

```diff
@@ -0,0 +1,45 @@
+package team05.integrated_feed_backend.module.post.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+import team05.integrated_feed_backend.common.enums.SocialMediaType;
+import team05.integrated_feed_backend.core.entity.BaseEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+
+@Entity
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
💭
액세스 레벨 설정해주셔도 좋을 것 같습니다. 
**참고자료**
https://velog.io/@kevin_/%EB%82%B4%EA%B0%80-NoargsConstructor-access-AccessLevel.PROTECTED%EB%A5%BC-%EC%99%9C-%EC%9E%91%EC%84%B1%ED%96%88%EC%9D%84%EA%B9%8C
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor(access = AccessLevel.PRIVATE)
```